src/components/CheckEmail/index.tsx

```diff
@@ -66,21 +80,39 @@ const CheckEmail = ({
     <Modal.Root className={cn(className, styles.root)} {...props}>
       <Modal.CloseButton aria-label="close signup modal" onClick={onClose} />
       <Modal.HeadingIcon className={styles.emailRoundedIcon}>
-        <Email className={styles.icon} width={20} height={20} />
+        <HyperPlayLogoIcon className={styles.icon} width={20} height={20} />
       </Modal.HeadingIcon>
       <Modal.Header>
         <Modal.Title>{i18n.title}</Modal.Title>
-        <Modal.Body>
-          {i18n.subtitle} <span className="text--semibold">{email}</span>
+        <Modal.Body className={'body-sm'}>
+          {i18n.subtitle} <span className="text--semibold">{email}</span>.{' '}
+          {i18n.clickLoginTitleOr}{' '}
+          <Button
+            type="link"
+            className={styles.reEnter}
+            onClick={onReEnterEmail}
+          >
+            {i18n?.reEnterEmail}.
```

Comment:
Same comment as I left on another PR:
Breaking up all the strings will mess up how it gets translated. Using Trans should let you use components while keeping the translation string together https://react.i18next.com/latest/trans-component 

We make the same mistake elsewhere but I'm trying to call it out when I see it

Reply:
nice catch, I'd completely forgotten I did the same here

Reply:
We don't have i18n next in the repo though 😅

Reply:
what do you think about creating a ticket for this?

Reply:
added here https://github.com/HyperPlay-Gaming/hyperplay-ui/issues/384